<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cooing</class>
 <widget class="QMainWindow" name="Cooing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cooing</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWebView" name="webView">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>481</width>
      <height>311</height>
     </rect>
    </property>
    <property name="url">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="typeBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>380</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendBtn">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>380</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Cooing.qrc">
      <normaloff>:/Resources/SendBtn.png</normaloff>:/Resources/SendBtn.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Cooing.qrc"/>
 </resources>
 <connections/>
</ui>
